templates/base/app/[locale]/page.tsx

```
@@ -0,0 +1,339 @@
+      borderRadius: 1,
+      bgcolor: 'rgba(255,255,255,0.08)',
+      border: '1px solid rgba(255,255,255,0.12)',
+      fontFamily: 'ui-monospace, SFMono-Regular, Menlo, Monaco, Consolas, monospace',
+      fontSize: 14
+    }}
+  >
+    {children}
+  </Box>
+)
+
+function HeroSection({}: { copied: boolean; handleCopy: (text: string) => void }) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)